wemake_python_styleguide/visitors/ast/function_empty_lines.py

```diff
@@ -30,9 +30,8 @@ def _is_target_line(self, token: tokenize.TokenInfo) -> bool:
         is_comment = False
         if stripped_token_line:
             is_comment = '#' in stripped_token_line[0]
```

Comment:
Shouldn't this be `is_comment = '#' == stripped_token_line[0]`?

Reply:
Yes, you're right. Thanks!